packages/cli-config-apple/src/tools/installPods.ts

```diff
@@ -10,27 +10,55 @@ import {
   runSudo,
 } from '@react-native-community/cli-tools';
 import runBundleInstall from './runBundleInstall';
+import path from 'path';
 
 interface PodInstallOptions {
   skipBundleInstall?: boolean;
   newArchEnabled?: boolean;
   iosFolderPath?: string;
+  platform: string;
+  root: string;
```

Comment:
This should be the location of the `xcodeproj` file. f that's the case, we are good to go.

Reply:
I don't think so, `root` is where `package.json` should be - based on this code:

https://github.com/facebook/react-native/blob/e41887e62fcaf5a8d6bfc2817f88298b49d1b7a5/packages/react-native/scripts/codegen/generate-artifacts-executor.js#L995